multicluster/values/values.go

```diff
@@ -40,7 +41,6 @@ type Values struct {
 // Gateway contains all options related to the Gateway Service
 type Gateway struct {
 	Enabled            bool              `json:"enabled"`
-	Replicas           uint32            `json:"replicas"`
```

Comment:
Why this removal?

Reply:
Ha, good spot. I thought we embedded the `Gateway` field so we'd have a conflict. I'll add back in.